Riot/Modules/Spaces/SpaceMembers/SpaceMembersCoordinator.swift

```diff
@@ -131,8 +131,51 @@ extension SpaceMembersCoordinator: SpaceMemberListCoordinatorDelegate {
     func spaceMemberListCoordinatorDidCancel(_ coordinator: SpaceMemberListCoordinatorType) {
         self.delegate?.spaceMembersCoordinatorDidCancel(self)
     }
+    
+    func spaceMemberListCoordinatorShowInvite(_ coordinator: SpaceMemberListCoordinatorType) {
+        guard let space = parameters.session.spaceService.getSpace(withId: parameters.spaceId), let spaceRoom = space.room else {
+            MXLog.error("[SpaceMembersCoordinator] spaceMemberListCoordinatorShowInvite: failed to find space with id \(parameters.spaceId)")
+            return
+        }
+        
+        spaceRoom.state { [weak self] roomState in
+            guard let self = self else { return }
+            
+            guard let powerLevels = roomState?.powerLevels, let userId = self.parameters.session.myUserId else {
+                MXLog.error("[RoomParticipantsInviteCoordinatorBridgePresenter] present: powerLevels room found")
```

Comment:
I'm not sure I understand this error message. Wondering if it is meant to be something like this?
```suggestion
                MXLog.error("[SpaceMembersCoordinator] spaceMemberListCoordinatorShowInvite: failed to find powerLevels for room")
```